modules/serverlist.js

```diff
@@ -0,0 +1,81 @@
+// Made by epsilon#3892
+// https://forum.thd.vg/members/16800/
+
+var config = require("../config.json");
+var request = require("request")
+
+module.exports = function(client) {
+	var module = {};
+
+	var channel, message;
+	channel = client.channels.get(config.server_list.channel);
+	if (!channel) return;
+	channel.fetchMessage(config.server_list.message).then(msg => {
+		message = msg;
+		loop();
+	}).catch(err => {
+		return; // Don't do anything if the message doesn't exist
+	});
+
+	function loop() {
```

Comment:
change the function name to something less vague. Maybe refresh or refreshServers.